Highway.h

```diff
@@ -2,8 +2,10 @@
 
 #include <vector>
 #include <algorithm>
```

Comment:
are you using anything from either of these two includes in this header file?
Don't include what you don't use.
- [ ] revise